### ISSUE TITLE
jhd1313m1 driver is missing the scrolling API

### ISSUE THREAD

**Issue by mattetti** [hybridgroup/gobot#207]

**Comment by mattetti** [comment]
I tried to implement this method:

```go
// Scroll scrolls the dispay to the right or the left based on the passed
// param.
func (h *JHD1313M1Driver) Scroll(leftToRight bool) error {

	if leftToRight {
		return h.connection.I2cWrite(h.rgbAddress, []byte{LCD_CMD, LCD_CURSORSHIFT | LCD_DISPLAYMOVE | LCD_MOVELEFT})
	}

	return h.connection.I2cWrite(h.rgbAddress, []byte{LCD_CMD, LCD_CURSORSHIFT | LCD_DISPLAYMOVE | LCD_MOVERIGHT})
}
```

Which is supposed to be a port of https://github.com/intel-iot-devkit/upm/blob/master/src/lcd/jhd1313m1.cxx#L109

Unfortunately this feature doesn't seem to work, I'm not getting an error but nothing happens and the backlit seems to be in a weird state where I can't set the rgb colors anymore.
I also tried on an arduino and on the edison. I tried to change the first byte we write, from `0x80` to `0x40`, it wasn't doing anything on the arduino and it returned an error on the edison.

**Comment by mattetti** [comment]
addressed in #209
